Query across multiple workspaces

francescomolfese opened the pull request:
Previously with Azure Log Analytics, you could only analyze data from within the current workspace and this limited your ability to query across multiple workspaces defined in your subscription.
Now you can query across multiple workspaces, providing a system-wide view of your data.

azurecla commented:
Hi __@saraclay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

davidwrede commented:
Hi @tfitzmac, can you take a look at this?

Thanks,
Dave

rmcmurray commented:
#sign-off

tfitzmac commented:
looks good to me

ggailey777 commented:
@rmcmurray or @tfitzmac if we still want this contribution can you work with @saraclay to fix the merge conflicts in her PR? Thanks.

ggailey777 commented:
@saraclay thanks for your attempt to contribute, but we need to close this because it looks like other changes were already made to fix this topic. We look forward to being able to use your contributions to Azure docs in the future.

francescomolfese committed:
Query across multiple workspaces

Previously with Azure Log Analytics, you could only analyze data from within the current workspace and this limited your ability to query across multiple workspaces defined in your subscription.
Now you can query across multiple workspaces, providing a system-wide view of your data.
